### DISCUSSION TITLE
Leaving Earth - Plan B

### DISCUSSION THREAD

**Discussion by kyrawertho** [colobot/colobot#1591] [General]
Hi, I just rediscovered the game after many years and although I have no trouble finishing the regular game, I can't even get through the first mission of the + missions since I get stuck here:

"Send an empty information to instruct the carrier to bring the black box back aboard your spaceship."

I can't find any help anywhere in the game or online. How can I figure out what to send?

**Comment by tomangelo2** [comment]
You need to send a message with empty name and any value. like that `send("", 0);`

Since there is nothing between quotation marks, the message is empty.

**Comment by kyrawertho** [comment]
Thanks, that's it. I was overcomplicating it 🤦
